zdm/scripts/run_mcmc.slurm

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#SBATCH --job-name=CRAFT
+#SBATCH --output=../mcmc/Hoffmann2023_CRAFT_no_191228.out
+#SBATCH --ntasks=20
+#SBATCH --time=24:00:00
```

Comment:
Does it actually take 24 hr? Looks like this has just been set to the max possible. Or... is the run time genuinely close to 24 hr? (e.g. 20 hr or so?)

Reply:
I run it with enough walkers / steps that it caps out at 24 hours... generally it is better to run with more steps for longer because you don't lose as much to burn-in, then your parallelization comes with more walkers.